src/App.tsx

```diff
@@ -1,27 +1,60 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setgoods] = useState<Good[]>([]);
```

Comment:
```suggestion
  const [goods, setGoods] = useState<Good[]>([]);
```
just FYI - use `camelCase`